js/jquery.nice-select.js

```diff
@@ -62,75 +62,76 @@
         .attr('tabindex', $select.attr('disabled') ? null : '0')
         .html('<span class="current"></span><ul class="list"></ul>')
       );
-        
+
       var $dropdown = $select.next();
       var $options = $select.find('option');
       var $selected = $select.find('option:selected');
-      
-      $dropdown.find('.current').html($selected.data('display') || $selected.text());
-      
+
+        $dropdown.find('.current').html($selected.data('display') || "<img src='" + $selected.attr('data-image') + "' width='50'>" + $selected.text());
```

Comment:
Could be a good change add this img only if data-image exists to avoid empty or 404 image errors